analyzers/src/SonarAnalyzer.Common/Rules/Utilities/SymbolReferenceAnalyzerBase.cs

```diff
@@ -52,6 +52,7 @@ public abstract class SymbolReferenceAnalyzerBase : UtilityAnalyzerBase<SymbolRe
         internal abstract SyntaxNode GetBindableParent(SyntaxToken token);
         protected abstract bool IsIdentifier(SyntaxToken token);
 
+        protected sealed override bool SkipAnalysisForTestProject => false;
```

Comment:
Can this default value be in the base class and overriden only when needed for those two?